docs/trex_release-notes.md

```diff
@@ -33,7 +36,17 @@ See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
    For information about the data model, see [The Tableau Data Model](https://help.tableau.com/current/pro/desktop/en-us/datasource_datamodel.htm){:target="_blank"}{:ref="noopener"}.
 
-* `DataValue` now has a `nativeValue` member. This member represents the raw native value as a JavaScript type, which is one of string, number, boolean, or Date object. Note that special values are returned as null. The `nativeValue` helps simplify error checking as all values will either be their native type value or null. The `nativeValue` exists for *ALL* `DataValue` objects, including those returned from parameters, filters, selections, and underlying or summary data.
+* `DataValue` now has a `nativeValue` member. This member represents the raw native value as a JavaScript type, which is one of string, number, boolean, or Date object. Note that special values are returned as null. The `nativeValue` helps simplify error checking as all values will either be their native type value or null. The `nativeValue` exists for *ALL* `DataValue` objects, including those returned from parameters, filters, selections, and underlying or summary data. Dates values are in UTC.
+
+* Added documentation for the [clearSelectedMarksAsync()]({{site.baseurl}}/docs/interfaces/worksheet.html#clearselectedmarksasync) method, which clears the selected marks in the current worksheet.
+
+Bugs fixed in this release:
+
+* Range filters now work correctly when the minimum or maximum values are equal to zero (0). Previously, calls to the `applyRangeFilterAsync()` method would ignore the `RangeFilterOptions` if the `min` or `max` properties were equal to zero (0).
+
+* The `isVisble` states for dashboard zones now work as expected. Prior to this release, if you used the  `setZoneVisibilityAsync()` method to hide a zone, the `isVisible` attribute for that zone wouldn't be updated to **false**. Consequently, the `isVisible` attribute would say that dashboard zone was visible when it was not.
```

Comment:
Thanks, @johnDance.  I'll update the notes and add the info about dashboard refresh and revise the `isVisible` bug with your suggestions.

Reply:
Now looks like this: 

* Previously, when a user was viewing a dashboard on Tableau Server or Tableau Online and an extension API call was denied because the user did not have permission for that functionality, the dashboard would be refreshed and extension reloaded. This caused the user to potentially lose state. Now the permission will be properly denied, but the dashboard and extension will not reload.

* The `isVisible` attribute for dashboard zones is now set properly to true or false when the extension is initialized.